zanata-war/src/main/java/org/zanata/adapter/OkapiFilterAdapter.java

```diff
@@ -172,6 +168,9 @@ public Resource parseDocumentFile(URI documentContent,
         RawDocument rawDoc =
                 new RawDocument(documentContent, "UTF-8",
                         net.sf.okapi.common.LocaleId.fromString("en"));
+        if (rawDoc.getTargetLocale() == null) {
+            rawDoc.setTargetLocale(net.sf.okapi.common.LocaleId.ENGLISH);
```

Comment:
Don't we assume in Zanata that all source docs are in English, if not specified otherwise?